lib/control/v1/conqueso.js

```diff
@@ -120,9 +120,19 @@ function Conqueso(app, storage) {
   }
 
   route.get((req, res) => {
+    const prop = req.params['0'].replace(new RegExp('/', 'g'), '');
```

Comment:
There definitely is and your way is preferable. I had to poop this out in a short period of time as it was blocking @storance-r7.